src/Instance/InstanceDetails/InstanceAdministrativeView/InstanceAdministrativeView.js

```diff
@@ -27,7 +28,23 @@ import {
 } from '../../../components';
 
 import StatisticalCodesList from './StatisticalCodesList';
-import { QUERY_INDEXES } from '../../../constants';
+import { QUERY_INDEXES, SOURCE_VALUES } from '../../../constants';
+
+const getSourceValue = (source) => {
+  if (!source || source === '-') {
+    return <NoValue />;
+  }
+
+  if (source === SOURCE_VALUES.CONSORTIUM_FOLIO) {
+    return <FormattedMessage id="ui-inventory.folioShared" />;
+  }
+
+  if (source === SOURCE_VALUES.CONSORTIUM_MARC) {
+    return <FormattedMessage id="ui-inventory.marcShared" />;
+  }
+
+  return source;
```

Comment:
Add `MARC` and `FOLIO` values to the SOURCE_VALUES, please
```suggestion
  return SOURCE_VALUES[source] || source;
```